packages/mui/src/BaseInputTemplate/BaseInputTemplate.tsx

```diff
@@ -79,7 +79,7 @@ export default function BaseInputTemplate<
         id={id}
         name={id}
         placeholder={placeholder}
-        label={displayLabel ? label || schema.title : false}
```

Comment:
Can you make the same change in the `material-ui` theme as well. The code is nearly identical